Bottom: c52933538d6cf22b5bf4ada59eabbf94db09a3bf
Top:    3df01f4f575b57c119c54048f61e5a2a716f7ab3
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-07-12 15:38:59 -0500

Refresh of gitlab-auth

---
diff --git a/roles/buildbot-master/defaults/main.yml b/roles/buildbot-master/defaults/main.yml
index c9a90fab..cc654992 100644
--- a/roles/buildbot-master/defaults/main.yml
+++ b/roles/buildbot-master/defaults/main.yml
@@ -45,8 +45,8 @@ container_env:
   BUILDBOT_WORKER_PTXDIST: 'worker-ptxdist'
   BUILDBOT_WORKER_PTXDIST_PASS: '{{ buildbot_worker_ptxdist_password }}'
   GITLAB_URI: 'https://{{ gitlab_hostname }}/'
-  GITLAB_CLIENT_ID: '{{ gitlab_buildbot_id }}'
-  GITLAB_CLIENT_SECRET: '{{ gitlab_buildbot_secret }}'
+  GITLAB_CLIENT_ID: '{{ gitlab_buildbot_auth_id }}'
+  GITLAB_CLIENT_SECRET: '{{ gitlab_buildbot_auth_secret }}'
   LDAP_URI: 'ldap://{{ openldap_hostname }}'
   LDAP_BIND_USER: 'cn=proxyagent,dc=novatech'
   LDAP_BIND_PW: '{{ openldap_proxyagent_password }}'
